app/components/block/BlockHistoryCard.tsx

```diff
@@ -40,10 +40,11 @@ const useQueryAccountFilter = (query: ReadonlyURLSearchParams): PublicKey | null
     return null;
 };
 
-type SortMode = 'index' | 'compute' | 'fee' | 'reservedCUs';
+type SortMode = 'index' | 'compute' | 'txn cost' | 'fee' | 'reservedCUs';
```

Comment:
this key should be one word, since it goes into the `UrlSearchParam` as `sortBy=txn%20cost`. simpler if key was just `txnCost`